main/static/css/style.css

```diff
@@ -0,0 +1,94 @@
+body {
```

Comment:
The name of the file is not good.
The name of the CSS file should be the same as the HTML and in the same directory with it, or in a directory for stylesheets, for better orientation.

Reply:
Fixed.